### ISSUE TITLE
[BUG] (short description)

### ISSUE THREAD

**Issue by razaci** [bigtreetech/BIGTREETECH-SKR-mini-E3#475]
I have a BIGTREETECH SKR Mini E3 V2.0 New Upgrade Control Board 32Bit with TMC2209 with a ANTCLABS BLTouch on an ENDER 3 PRO [firmware-bltouch-for-z-homing.bin]

I have installed leaving the z stop plugged in. I installed the bltouch on the single pin area together with the 3 wires and 2 wires together with them plugged in correctly as in photo. (https://github.com/bigtreetech/BIGTREETECH-SKR-mini-E3/blob/master/hardware/BTT%20SKR%20MINI%20E3%20V2.0/Hardware/BTT%20SKR%20MINI%20E3%20V2.0-PIN.pdf). I am running firmware-bltouch-for-z-homing.bin firmware.

I have downloaded and left firmware.bin lowercase, flashed, and reset eprom. (least 6 times at this point)

I click auto level and it goes to center of board and drops to board. Right on, perfectly, and the bltouch is what it seems is what's telling it to stop. It drops down and hits the board, stops the printer, perfectly.

I  then click level board and it goes to the original front left corner and drops to board perfectly, doing as it had in center, then it says probing point 1/25  then drops down a couple times, stops with the following message on screen.

[[ ENDSTOP    Z  ]]

I have not modified any file. I did install VB and IO and downloaded configuration and have it open but didn't modify or compile. I have not changed anything. I have reset the eprom each flash and all my values. I just cant get the bltouch message to stop saying endstop z.

So then I decided to print because the board is level... I click print, bed warms, extruder warms, message on screen switches to "printing" and immediately the board temp starts dropping E temp starts dropping and they will go to zero, no print. notta. Just sits there shutting itself down.

**Comment by razaci** [comment]
update. when running the firmware-bltouch.bin resetting eprom and settings. auto home goes to front left corner and sets on bed perfectly. 

Bed level raised the printer head way up in the air and starts testing for the bed. uh..... why? So I press Z home and it drops perfectly to the bed, perfect. I click bed level and it freaking raises way up and tries to feel for the bed..... what? help!!!

Then since its level, I click print using this firmware and the bed heats up, the E heats up, but this time instead of dropping the heat on the bed and the E it just sits there in the corner ready to print on screen message says "printing..." but the extruder, the motors, nothing... its all just not printing. 

Which firmware should I be running and how do I fix whichever problem in that Ive listed firmware? help!!!!

**Comment by razaci** [comment]
i have the ANTCLABS BLTOUCH 3.1 and SKR mini V2 with TFT 35 on an Ender 3 Pro.

I have the same error when trying to enter home offset on marlin mode, Error too far!

Also when i try to jog my axes from the touch menu it does not work, and in Marlin mode it does normally.

I have managed to set the offsets for the bed in the Probe offset menu, but it seems that for the Z it does not work as expected.

I have connected both the Z-end Stop and BL Touch to the board as per the pinout described, everything is working correctly, i can test, i can use all the commands.
When i try to Auto Bed Level it homes first using the end stops on all X and Y then goes to the center, homes Z once stays down, then homes Z again and lifts Z 10 mm. and goes to the X and Y coordinates, i have set in the Probe Offset menu. And then starts probing, resulting in error.

If i try to change the offset to Z-10 for example then the lift after homing will be 20 mm. but if i set it to 9.8 for example, it will still lift to 10 mm.

I currently have the firmware-bltouch.bin on the printer.

Thank you kindly.

**Comment by razaci** [comment]
ok. this is what I have done. 

I have installed the irmware-bltouch.bin and firmware-bltouch-for-z-homing.bin clearing after each install as like new. I do not understand what is wrong. so, I installed the basic firmware.bin from the bottom of the list on github..... and the printer works perfectly. 

I have a bigtreetech v2.0 board, so I searched for other boards yes I could go up or use another brand but everyone has so much data fixed on 2.0 the board cant be the problem.

I have the ANTCLABS BLTouch and I paid like 40 on amazon for it thinking matched from their website it was the correct part when suggested. This has to be my problem. So I have ordered the USA 3D BLTouch V3.0 Auto Leveling Sensor Kit BL Touch Sensor For Ender 3 Pro and when it arrives will install using the z stop plug in place of my z stop switch. I will then use the correct  bin and let you guys know if that corrects this.

Nothing is wrong with the printer, it prints amazing prints now with silent bigtreetech v 2.0 board, only when using the advanced bin files and the bltouch from antclabs do I think I present the above problems for myself.

I hope. I will let you know.
